js/sign.js

```diff
@@ -0,0 +1,114 @@
+var check = document.getElementById("check");
+var check1 = document.getElementById("check1");
+var check2 = document.getElementById("check2");
+var check3 = document.getElementById("check3");
+
+var nameInput=document.getElementById("name_input");
+var emailInput = document.getElementById("email_input");
+var passwordInput = document.getElementById("password_input");
+var passwordInput2 = document.getElementById("password_input2");
```

Comment:
No need for defining those variables as `var`, define them as `let`. By using var you're polluting the window object.

Reply:
done